include/range/v3/detail/config.hpp

```diff
@@ -389,6 +389,14 @@ namespace ranges
 #define RANGES_RE_ENABLE_WARNINGS
 #endif
 
+
+#if __has_attribute(no_sanitize)
+#define RANGES_INTENDED_UNSIGNED_OVERFLOW \
```

Comment:
> I apologize for nitpicking at semantics here

No need to apologize, I think this discussion is worth having. 

> have repeatedly chosen to use unsigned types to mean non-negative 

I wouldn't really blame them that much. They are called `unsigned` after all. Its deviously misleading.

> I suppose I can live with `RANGES_INTENDED_WRAP_AROUND`.

I think I would go for `RANGES_INTENDED_UNSIGNED_WRAP_AROUND`, since we don't intend signed integers to wrap around.  `RANGES_INTENDED_UNSIGNED_OVERFLOW` is a bad name though, I don't want to keep that.